apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -111,6 +115,29 @@ async function canManageTickets (args) {
             return ticket.client === user.id
         }
     } else {
+        // TODO: DOMA-10832 add check employee organization in Ticket access
+        if (isBulkRequest && operation === 'update') {
+            if (itemIds.length !== uniq(itemIds).length) return false
+            if (itemIds.length !== originalInput.length) return false
+
+            if (!originalInput.every((updateItem) => {
+                for (const key in updateItem.data) {
+                    if (!BULK_UPDATE_ALLOWED_FIELDS.includes(key)) return false
+                }
+                return true
+            })) return false
+
+            const tickets = await find('Ticket', {
+                id_in: itemIds,
+                deletedAt: null,
+            })
+
+            const ticketOrganizationIds = uniq(tickets.map(ticket => get(ticket, 'organization', null)))
```

Comment:
Why not let only update stuff from one org?

Reply:
There may be a case when the CC employees want to remove someone and at the same time reassign all of his requests to another CC employee. The employee being removed may have requests from different organizations, which is why this operation will be rejected